node/tests/swamp/config.go

```diff
@@ -9,19 +9,19 @@ import (
 	"github.com/celestiaorg/celestia-node/core"
 )
 
-// Config struct represents a set of pre-requisite attributes from the test scenario
-type Config struct {
+// InfraComps struct represents a set of pre-requisite attributes from the test scenario
+type InfraComps struct {
```

Comment:
What does `Comps` stand for?

Reply:
Thanks for the suggestions. Changed to suggested ones